src/main/java/uk/gov/hmcts/reform/civil/service/JudgementService.java

```diff
@@ -20,6 +20,9 @@ public class JudgementService {
 
     private static final String JUDGEMENT_BY_COURT = "The Judgement request will be reviewed by the court,"
         + " this case will proceed offline, you will receive any further updates by post.";
+
+    private static final String JUDGEMENT_BY_COURT_NOT_OFFLINE = "The Judgement request will be processed and a Country"
```

Comment:
The first "Judgment" is not with capital "J" in the example.
"County" instead of "Country"